optimum/exporters/openvino/__main__.py

```diff
@@ -88,6 +88,18 @@ def infer_task(
                     token=token,
                     library_name=library_name,
                 )
+                # phi-4-multimodal has more common with multimodal, however pipeline_tag defined as automatic-speech-recognition on hub
+                if "automatic-speech-recognition" in task and library_name == "transformers":
```

Comment:
is it needed because for quantization we are checking for the task to match like in :
 https://github.com/huggingface/optimum-intel/blob/3910592c0cdb9f11ce0d773cc3ec14d7713b8522/optimum/exporters/openvino/convert.py#L98

in all cases why not checking for the architecture instead [MULTI_MODAL_TEXT_GENERATION_MODELS](https://github.com/huggingface/optimum-intel/blob/3910592c0cdb9f11ce0d773cc3ec14d7713b8522/optimum/exporters/openvino/utils.py#L220) ?

Reply:
not sure tha tI understand you.
when we use quantization with dataset via optimum-cli, we use task to determine which class should be used for model loading here https://github.com/huggingface/optimum-intel/blob/main/optimum/commands/export/openvino.py#L406

phi-4-multimodal model card contains pipeline_tag as automatic-speech-recognition here https://huggingface.co/microsoft/Phi-4-multimodal-instruct

as the result instead of multimodal model loading we will try to create speech recognition model

Reply:
OK I see this is only needed for the CLI + data aware quantization (it could also make sense to only set it default there)

Reply:
moved to command